### DISCUSSION TITLE
在Android编libpag的Android库出错，这个问题需要怎么解决呢？

### DISCUSSION THREAD

**Discussion by a1198375223** [Tencent/libpag#1616] [Q&A] [answered]
报错信息如下：
![image](https://github.com/Tencent/libpag/assets/24283635/1f1ad224-d5bc-4791-8bd2-6af16d58b3c7)

Android编译环境如下：
![image](https://github.com/Tencent/libpag/assets/24283635/c44bce3c-541b-4207-8b03-a227882ce8a0)

**Comment by kevingpqi123** [comment, marked as answer]
可以看下 README，按照上面的流程先拉取代码及第三方库，不要放过每一个报错，先确保 android 侧的 demo 工程能够正常运行起来，然后再编译库文件

**Reply by a1198375223** [reply]
我是按照上面的wiki流程来编译的，但是还是在运行的时候会出上面的错误，不知道您通过上面的配置流程在MacOS中编译是否有遇到相同的问题呢？

**Reply by kevingpqi123** [reply]
没有，操作的过程中不要放过每一个报错信息，可以将完整的执行过程中放在文件中发出来看下
